CHANGELOG.md

```diff
@@ -11,7 +13,7 @@
 - Upload coverage reports to Codecov. (#61)
 - Deploy contracts in tests explicitly with hardhat or truffle fixture. (#59)
 - Add the ability to deploy without truffle fixture. (#58)
-- Sponsor match orders. (#57, #60)
+- Sponsor match orders. (#57, #60, #79)
```

Comment:
why add it here ? we putted `Refund sponsor on `claim`. (#77)`

Reply:
Ok updated!